### ISSUE TITLE
how to fix this?

### ISSUE THREAD

**Issue by cyrstem** [CLOUDS-Interactive-Documentary/CLOUDS#41]
every time  try to build   clouds i get  this

clang: error: no such file or directory: '/Volumes/GhOsT/Creative Coding/OF/openFrameworks/libs/openFrameworks/sound/ofBaseSoundPlayer.cpp'
clang: warning: -lpthread: 'linker' input unused
Command /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang failed with exit code 1
with exit code 1
how can i fixed

**Comment by obviousjim** [comment]
what eversion of openFrameworks are you buliding against?

On Fri, Mar 13, 2015 at 1:12 PM, cyrstem <notifications@github.com> wrote:

> every time try to build clouds i get this
>
> clang: error: no such file or directory: '/Volumes/GhOsT/Creative
> Coding/OF/openFrameworks/libs/openFrameworks/sound/ofBaseSoundPlayer.cpp'
> clang: warning: -lpthread: 'linker' input unused
> Command
> /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang
> failed with exit code 1
> with exit code 1
> how can i fixed
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CLOUDS-Interactive-Documentary/CLOUDS/issues/41>.
>



-- 
- James

**Comment by cyrstem** [comment]
feature-64bit branch the one from the read me setup ? follow the steps but can't build ...

**Comment by obviousjim** [comment]
We switched to CLOUDS branch now

what operating system and version?


On Fri, Mar 13, 2015 at 1:26 PM, cyrstem <notifications@github.com> wrote:

> feature-64bit branch the one from the read me setup ? follow the steps but
> can't build ...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CLOUDS-Interactive-Documentary/CLOUDS/issues/41#issuecomment-79159375>
> .
>



-- 
- James

**Comment by cyrstem** [comment]
so what should i do ? just updated
to  Yosemite 10.10.2

**Comment by obviousjim** [comment]
Ahh,

We don't have a project file that works yet for 10.10 :( the builds run
fine but the Xcode version has errors. You'll have to build from an older
10.9 system for the time being.

On Fri, Mar 13, 2015 at 3:16 PM, cyrstem <notifications@github.com> wrote:

> so what should i do ? just updated
> to Yosemite 10.10.2
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CLOUDS-Interactive-Documentary/CLOUDS/issues/41#issuecomment-79270611>
> .
>



-- 
- James

**Comment by neilmendoza** [comment]
What about switching the Base SDK to 10.9 in the build settings?

--
www.neilmendoza.com
twitter.com/neilmendoza
+1 424 229 4538

**Comment by obviousjim** [comment]
Thanks Neil - Does it work for you?

On Sun, Mar 15, 2015 at 2:26 PM, Neil Mendoza <notifications@github.com>
wrote:

> What about switching the Base SDK to 10.9 in the build settings?
>
> --
> www.neilmendoza.com
> twitter.com/neilmendoza
> +1 424 229 4538
>
> On 14 March 2015 at 08:30, James George <notifications@github.com> wrote:
>
> > Ahh,
> >
> > We don't have a project file that works yet for 10.10 :( the builds run
> > fine but the Xcode version has errors. You'll have to build from an older
> > 10.9 system for the time being.
> >
> > On Fri, Mar 13, 2015 at 3:16 PM, cyrstem <notifications@github.com>
> wrote:
> >
> > > so what should i do ? just updated
> > > to Yosemite 10.10.2
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> >
> https://github.com/CLOUDS-Interactive-Documentary/CLOUDS/issues/41#issuecomment-79270611
> > >
> > > .
> > >
> >
> >
> >
> > --
> > - James
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <
> https://github.com/CLOUDS-Interactive-Documentary/CLOUDS/issues/41#issuecomment-80534984
> >
>
> > .
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CLOUDS-Interactive-Documentary/CLOUDS/issues/41#issuecomment-81188975>
> .
>



-- 
- James

**Comment by cyrstem** [comment]
nop gives me a new error  /Volumes/GhOsT/Creative Coding/OF/of_v0.8.4_osx_release/addons/ofxSubtitles/src/ofxSubtitles.h:14:10: 'ofxTimecode.h' file not found witch is strange because  i can see it right there .

**Comment by obviousjim** [comment]
perhaps re-add that addon to to the project file?

On Sun, Mar 15, 2015 at 2:40 PM, cyrstem <notifications@github.com> wrote:

> nop gives me a new error /Volumes/GhOsT/Creative
> Coding/OF/of_v0.8.4_osx_release/addons/ofxSubtitles/src/ofxSubtitles.h:14:10:
> 'ofxTimecode.h' file not found witch is strange because i can see it right
> there .
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CLOUDS-Interactive-Documentary/CLOUDS/issues/41#issuecomment-81192951>
> .
>



-- 
- James

**Comment by cyrstem** [comment]
neither ill check in other system

**Comment by obviousjim** [comment]
Hi,

Xcode 6 should now be fixed. Could you give this another try?

**Comment by cyrstem** [comment]
ok  ill try now   thanks

**Comment by cyrstem** [comment]
nop not working either

**Comment by obviousjim** [comment]
Hey there,

can you post more info about what errors your getting?

On Tue, Mar 17, 2015 at 1:01 PM, cyrstem <notifications@github.com> wrote:

> nop not working either
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CLOUDS-Interactive-Documentary/CLOUDS/issues/41#issuecomment-82481101>
> .
>



-- 
- James

**Comment by cyrstem** [comment]
same error  i'll try downloading again

**Comment by obviousjim** [comment]
its' most likely a problem with your addon configurations. Did you run the
cloneaddons script successfully? if it's the ofxSubtitles error you posted
it most likely is just missing that addon.

On Tue, Mar 17, 2015 at 1:08 PM, cyrstem <notifications@github.com> wrote:

> same error i'll try downloading again
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CLOUDS-Interactive-Documentary/CLOUDS/issues/41#issuecomment-82484238>
> .
>



-- 
- James

**Comment by cyrstem** [comment]
yeah  thats the weird thing  the sh  run  just fine,  but ill try form the start  again see  what i did wrong 
ill  le t you know how it goes really looking forward to see this running

**Comment by prisonerjohn** [comment]
@cyrstem Can you confirm you're using the OF version found in [this repo](https://github.com/CLOUDS-Interactive-Documentary/openFrameworks/tree/CLOUDS), and that you're on the CLOUDS branch?

I'm asking again because the path you mention in your comment above indicates otherwise:
"/Volumes/GhOsT/Creative Coding/OF/**of_v0.8.4_osx_release**/addons/ofxSubtitles/src/ofxSubtitles.h"

**Comment by cyrstem** [comment]
yeah  i change branch from that one  but i got the same error,  i  try everything from the start again   ill take a  while to download    but  i keep trying hehe

**Comment by obviousjim** [comment]
cool, thanks for your persistence. It's always good to make sure things are
building for other people.

Let us know how it goes.

On Tue, Mar 17, 2015 at 1:20 PM, cyrstem <notifications@github.com> wrote:

> yeah i change branch from that one but i got the same error, i try
> everything from the start again ill take a while to download but i keep
> trying hehe
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CLOUDS-Interactive-Documentary/CLOUDS/issues/41#issuecomment-82487572>
> .
>



-- 
- James

**Comment by gorkacortazar** [comment]
Guys, I was trying to replicate this error, and i found a few of errors in the cloneaddons script.
1) ofxDepthKit reports this error: "ERROR: Repository not found. fatal: Could not read from remote repository."
2) ofxMSATimer, ofxObjLoader, ofxTimecode and ofxTimeline are pulled to the root of OF, not to the addons folder, that may cause @cyrstem reported error

**Comment by obviousjim** [comment]
Ahh good call! that's an error:

please replace that with this

https://github.com/CLOUDS-Interactive-Documentary/ofxRGBDepth

On Tue, Mar 17, 2015 at 1:48 PM, Gorka Cortázar <notifications@github.com>
wrote:

> Guys, I was trying to replicate this error, and i found a few of errors in
> the cloneaddons script.
> 1) ofxDepthKit reports this error: "ERROR: Repository not found. fatal:
> Could not read from remote repository."
> 2) ofxMSATimer, ofxObjLoader, ofxTimecode and ofxTimeline are pulled to
> the root of OF, not to the addons folder, that may cause @cyrstem
> <https://github.com/cyrstem> reported error
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CLOUDS-Interactive-Documentary/CLOUDS/issues/41#issuecomment-82495971>
> .
>



-- 
- James

**Comment by gorkacortazar** [comment]
I finally managed to get the application compile and runs in OSX. you have to do the following steps:
1) Apart from ofxRGBDepth that needs to replace ofxDepthKit, the cloneaddons.sh script also misses ofxDelaunay.
2) Some of the VS seems to come form a private repository, specifically CloudsVisualSystemLIA, CloudsVisualSystemProcess11 and CloudsVisualSystemProcess18 (by Casey Reas this last two?).
3) I removed them from the project the files in the VisualSystemsPrivate folder
4) Commented the related includes in the file CloudsVisualSystemManager.cpp; also in the struct with the clases mapping i replaced the class creation for those VS with CloudsVisualSystemHistogram, is probably is a better idea to create an empty VS for those?.

I also rename a namespace in the ofxSubtitles, i think that there is a typo of an ofxTimeline related class?

After that the application compiles and run (at least the general version) throwing the expected error because it misses the movies and the file that defines where they are located.  I get a lot of errors of presets, media and from CloudsFCPParser, also expected from the core with no media, but the last one I am not that sure of what it is for.

One think that i didn't understand is that the time line is no accepting the play command, it may be related to the missing media and parsing errors?

**Comment by obviousjim** [comment]
@gorkacortazar great work! I'm sure you are the first to actually get this building.

I will create an issue for creating a build target for the public version that removes the private repos.

We are currently working on the public release,and will be making the media available for the open source version to link against.
